motioneye/handlers/config.py

```diff
@@ -214,6 +217,11 @@ def set_main_config(ui_config):
             reload = False
             restart = False
 
+            if lang != old_lang:
+                logging.debug('lang changed, restart needed')
+                # kill myself to force restart. Not very elegant...
+                os.kill(os.getpid(), 9)
```

Comment:
Not very elegant as well, but indeed seems to be SIGKILL is the only signal where systemd triggers an automated restart. We may want to implement a specific signal or exit code and a related `Restart=` to the systemd unit.